### PR TITLE
Included react-router and placeholder pages

### PR DESCRIPTION
**Pull request by shahbaz4595**
Description: Added react-router for basic routing and created placeholder pages.

Site Preview: https://deploy-preview-2--bleedbluestore.netlify.app/

### CODE REVIEW

**Review by Developers-codz — APPROVED** [review]
good folder structure and readable code. Looks good to merge

File: bb-store/src/App.js [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
+        <Header />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/shop" element={<Shop/>} />
+          <Route path="/wishlist" element={<Wishlist/>} />
+          <Route path="/cart" element={<Cart/>} />
+          <Route path="/signup" element={<SignUp/>} />
+          <Route path="/login" element={<Login/>} />
+          <Route path="/profile" element={<UserProfile/>} />
+          <Route path="/*" element={<NotFound/>} />
```
Comment (arjun-computer-geek) [inline bb-store/src/App.js:17]: This is great. I like this.

File: bb-store/src/App.js [hunk 2]
```diff
@@ -1,11 +1,23 @@
-import { Header, Footer, Home } from "./BleedBlueStore";
+import { Header, Footer, Home, Shop, Wishlist, Cart, SignUp, Login, UserProfile, NotFound } from "./BleedBlueStore";
```
Comment (arjun-computer-geek) [inline bb-store/src/App.js:1]: This approach is also good.

**Review by arjun-computer-geek — APPROVED** [review]
All looks good. Great code structure. Cleaner code. 👍👍👍

**Comment by shahbaz4595** [comment]
thanks @Developers-codz and @arjun-computer-geek for the feedback and approval.

**Pull request merged**